toolchain/check/testdata/var/no_prelude/decl.carbon

```diff
@@ -15,16 +15,16 @@ fn Main() {
 // CHECK:STDOUT: --- decl.carbon
 // CHECK:STDOUT:
 // CHECK:STDOUT: constants {
-// CHECK:STDOUT:   %Main: type = fn_type @Main [template]
+// CHECK:STDOUT:   %Main.type: type = fn_type @Main [template]
 // CHECK:STDOUT:   %.1: type = tuple_type () [template]
-// CHECK:STDOUT:   %struct: Main = struct_value () [template]
+// CHECK:STDOUT:   %Main: <type of Main> = struct_value () [template]
```

Comment:
I'm a bit sad that we end up with two ways to name the same thing (`%Main.type` and `<type of Main>`).

Reply:
I'll send a separate PR to address that.